tests/test_mission_phase_race_condition.py

```
@@ -0,0 +1,200 @@
+        """Test that force_safe_mode can interrupt ongoing transitions."""
+        state_machine = StateMachine()
+        results = []
+        
+        def slow_transition():
+            """Simulate a slow transition."""
+            try:
+                state_machine._is_transitioning = True
+                time.sleep(0.15)
+                state_machine._is_transitioning = False
+                results.append("normal_completed")
+            except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)